views/scan.hbs

```diff
@@ -15,15 +15,7 @@
       {{else}}
         <h2 class="headline scan-results-headline">{{{ fluentNestedBold "scan-results-known-breaches" breachCount=foundBreaches.length }}}</h2>
         <button id="new-scan-page"
-          class="open-oauth ga-growth-ping {{#if experimentBranchB}} temp-marketing-btn-blue experiment {{else}} blue-link temp-marketing-alert-me-link {{/if}}"
-          {{#if experimentBranchB}}
-            {{> analytics/fxa id="fx-monitor-alert-me-blue-btn-top" }}
-          {{else}}
-            {{> analytics/fxa id="fx-monitor-alert-me-blue-link" }}
-          {{/if}}
-          data-event-category="Alert me about new breaches">
-            {{ getString "alert-about-new-breaches" }}
-          </button>
+          class="open-oauth blue-link temp-marketing-alert-me-link" {{> analytics/fxa id="fx-monitor-alert-me-blue-link" }} data-event-category="Alert me about new breaches"> {{ getString "alert-about-new-breaches" }} </button>
```

Comment:
This is the primary patch that needed to be reverted.

Reply:
@lesleyjanenorton - look good?

Reply:
looks good and runs fine locally